corehq/apps/products/fixtures.py

```diff
@@ -52,8 +53,19 @@ def get_products():
                 key=lambda product: product.code
             )
 
-        return simple_fixture_generator(
+        fixture_nodes = simple_fixture_generator(
             restore_user, self.id, "product", PRODUCT_FIELDS, get_products, restore_state.last_sync_log
         )
+        if not fixture_nodes:
+            return []
+
+        if restore_state.params.openrosa_version and restore_state.params.openrosa_version < '2.1':
+            # Don't include index schema when openrosa version is specified and below 2.1
```

Comment:
I see. I think it still makes sense to do 
```py
if not restore_state or openrosa_version >= OPENROSA_VERSION_MAP['INDEXED_PRODUCTS']:
...
```
You can also write a comment next to where you define that constant saying that this will also be applied if there is _no_ version, that way we keep everything in one place.

I'm curious when an openrosa version isn't sent down, and why you'd add the index in those instances?